### PR TITLE
Lesson 03-04 Bash, Make

### PR DESCRIPTION
**Pull request by danismi715**
- Add command to input folder name
- Add command to create folder with entered name
- Add command to search files with extension .c and .h and copy this files to created folder
- Add command to archive this folder
- Add command to remove this folder

@a-vodka Please check my PR.

### CODE REVIEW

**Comment by a-vodka** [comment]
- Directory name and its path are defined by the task.
- You should check existence of dirs

**Comment by danismi715** [comment]
- Add a check for the existence of a folder(directory)
- Add a permanent path (tmp/guess_a_number and release)
- Add Makefile for "lesson-02" project

File: 01-Git_Bash_Make/script [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#! /bin/sh
+
+#Create folders tmp/guess_a_number/
+mkdir -p tmp/guess_a_number
```
Comment (a-vodka) [inline 01-Git_Bash_Make/script]: please, define /tmp/guess_a_number as variable

Comment (danismi715) [reply 01-Git_Bash_Make/script]: Fixed it.

**Pull request merged**